pyscriptjs/examples/simple_clock.html

```diff
@@ -41,6 +47,8 @@
     else:
       out3.clear()
 
+# close the global PyScript pyscript_loader
+pyscript_loader.close()
```

Comment:
Gotcha. Yeah, I wasn't actually sure where to put it. Didn't want to add a new one just for the loader but actually it may just make sense...

So, your proposal, is to add a separate demo showing how to control the loader, right?

Reply:
yes. Either that, or no demo at all :)